react-main/src/app/components/TopbarGame/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component, Fragment } from 'react';
+import styles from './styles.scss';
+import user from '../../../images/user.svg';
+import logout from '../../../images/singout.svg';
+
+class TopbarGame extends Component {
+  render() {
+    return (
+      <div className={styles.topbar}>
+        <img src={user} className={styles.profile} alt="go-profile" onClick={this.props.handleProfile}/>
```

Comment:
alts should be something that the user can see and understand if an image doesn't load. I'd put something like "Profile", so if the image that represents the profile doesn't load, the user understands that that represents the profile. No camelcase, hyphens, etc, just normal text.